### PR TITLE
Web APIs: add HashChangeEvent

### PR DESCRIPTION
**Pull request by caugner**
[HashChangeEvent](https://developer.mozilla.org/docs/Web/API/HashChangeEvent)

### CODE REVIEW

File: api/HashChangeEvent.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
+{
+  "api": {
+    "HashChangeEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HashChangeEvent",
+        "support": {
+          "webview_android": {
+            "version_added": "2.2"
```
Comment (jpmedley) [inline api/HashChangeEvent.json]: Change this to `true`. Webview follows Chrome version numbers, not Android version numbers. (It's probably 5, but I can't confirm something that old, so `true`.)

File: api/HashChangeEvent.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+          },
+          "chrome": {
+            "version_added": "5"
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/HashChangeEvent.json]: This should be `true`.

File: api/HashChangeEvent.json [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HashChangeEvent/oldURL",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HashChangeEvent.json]: `true` for all 3 flavors of Chrome.

File: api/HashChangeEvent.json [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HashChangeEvent.json]: `true` for both flavors of Opera.

File: api/HashChangeEvent.json [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HashChangeEvent/newURL",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HashChangeEvent.json]: `true` for all 3.

File: api/HashChangeEvent.json [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HashChangeEvent.json]: `true` for both.

**Comment by caugner** [comment]
I have just incorporated the suggested changes from @jpmedley (thank you!), so this is ready to be merged from my side.

**Pull request merged**